en/config_vtol/vtol_quad_configuration.md

```diff
@@ -59,14 +58,16 @@ Parameter: [VT_B_TRANS_THR](../advanced_config/parameter_reference.md#VT_B_TRANS
 Generally back-transition throttle can be set to 0 since forward thrust is not (in most cases) desirable.
 If the motor controller supports reverse thrust however, you can achieve this by setting a negative value.
 
-#### Forward Transition Pusher/Puller Ramp-up Time
+#### Forward Transition Pusher/Puller Slew Rate

```

Comment:
FWIW I _think_ it is better to be precise here. What was there was more "readable" to me - i.e. I understood it without having to read what slew rate means.